keyboards/makrosu/Makefile

```diff
@@ -0,0 +1,3 @@
+ifndef MAKEFILE_INCLUDED
```

Comment:
This file should be removed, as it's deprecated.

Reply:
ok done